app/proguard-rules.pro

```diff
@@ -27,9 +27,7 @@
   <fields>;
 }
 
--keepnames class androidx.navigation.fragment.NavHostFragment
--keepnames class com.willowtree.vocable.room.Category
--keepnames class com.willowtree.vocable.room.Phrase
+-keepnames class * extends android.os.Parcelable
```

Comment:
could this retain anything parcelable even in dependencies and stuff? I'm not an app size hound but we should check that apk size doesn't go up 10MB or anything after this

Reply:
The nice thing about `keepnames` is that all it does is preserve names of classes that **weren't** removed during shrinking. It doesn't keep any classes that otherwise would have been removed during shrinking.

See: https://www.guardsquare.com/manual/configuration/usage#keepoptions